### ISSUE TITLE
1.4.0-pre-6: UnboundLocalError: local variable 'texts' referenced before assignment with "-u" option

### ISSUE THREAD

**Issue by valsamis-d** [Aetopia/NVIDIA-Driver-Downloader#11]
@Aetopia hi,

Here's an issue I came across with 1.4.0-pre-6:

![image](https://user-images.githubusercontent.com/17590539/180408511-ae25548c-e75c-45eb-b491-1e87f91dd294.png)

Contents of "%TEMP%\nvddl.log"

```
INFO: Namespace: Namespace(help=False, list=True, download=None, extract=None, update=False, studio=False, standard=False, full=False, setup=False, components=[], output='F:\\Temp\\DVALSA~1', flags=[], no_stdout=False)
INFO: Arguments: ['-ls']
INFO: Getting Driver Versions...
INFO: Selected Driver Type: Game Ready
INFO: Selected Driver Type: DCH
INFO: Detected: GP107GLM [Quadro P1000 Mobile]
INFO: API Link Generated: https://www.nvidia.com/Download/processFind.aspx?psid=543&pfid=874&osid=57&lid=1&whql=1&ctk=0&dtcid=1
INFO: Finished!
```

Contents of "%TEMP%\nvddl_traceback.txt"

```
Traceback (most recent call last):
  File "F:\Temp\DVALSA~1\ONEFIL~2\main.py", line 19, in main
  File "F:\Temp\DVALSA~1\ONEFIL~2\core\cli.py", line 89, in cli
  File "F:\Temp\DVALSA~1\ONEFIL~2\core\cli.py", line 151, in parse
  File "F:\Temp\DVALSA~1\ONEFIL~2\core\functions.py", line 247, in update
UnboundLocalError: local variable 'texts' referenced before assignment
```

Dimitrios

**Comment by Aetopia** [comment]
Well this is a quick and easy fix.

**Comment by valsamis-d** [comment]
Fixed in 1.4.0-pre-7. Thank you @Aetopia :)
